<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>A52Dialog</class>
 <widget class="QDialog" name="A52Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter A5/2 key</string>
  </property>
  <widget class="QLabel" name="frameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter frame:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0x21</string>
   </property>
  </widget>
  <widget class="QLabel" name="keyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter keys:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0x00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xfc</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
